bench/config/lets_encrypt.py

```diff
@@ -86,7 +86,7 @@ def run_certbot_and_setup_ssl(site, custom_domain, bench_path, interactive=True)
 def setup_crontab():
 	from crontab import CronTab
 
-	job_command = '/opt/certbot-auto renew -a nginx --post-hook "systemctl reload nginx"'
+	job_command = '{path} renew --nginx -q --renew-hook "systemctl reload nginx"'.format(path=get_certbot_path())
```

Comment:
Feel like this should check which service is being used, like `bench.utils.service` does

also, f-string